css.css

```diff
@@ -0,0 +1,170 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  body{
+    
+    height: auto;
+  }
+
+  img {
+    max-width: 100%;
+    max-height: 100%;
+}
+
+  .container {
+    display: grid;
+    grid-template-columns: 0.1fr 1.8fr;
+    grid-template-rows: 0.1fr 1.8fr;
+    gap: 0px 0px;
+    grid-template-areas:
+      "sidebar navbar"
+      "sidebar content_box";
+  }
+  .sidebar { grid-area: sidebar; }
+  .content_box { grid-area: content_box; }
```

Comment:
try not to nest properties like that. write expanded code:
`.content_box { [new line]
  grid-area: content_box; [new line]
}`